Documentation/ceph-quickstart.md

```diff
@@ -24,6 +24,7 @@ If you are using `dataDirHostPath` to persist rook data on kubernetes hosts, mak
 If you're feeling lucky, a simple Rook cluster can be created with the following kubectl commands. For the more detailed install, skip to the next section to [deploy the Rook operator](#deploy-the-rook-operator).
 ```
 cd cluster/examples/kubernetes/ceph
+kubectl create -f common.yaml
```

Comment:
`common.yaml` is new in master and was not included with 0.9, so this shouldn't need to be added. It only applies to the soon-to-be 1.0 release